backend/inspirehep/records/marshmallow/jobs/base.py

```diff
@@ -18,22 +19,15 @@
 
 
 class JobsRawSchema(RecordBaseSchema):
-    class Meta:
-        exclude = ("can_edit",)
-
     def __init__(self, *args, **kwargs):
         super().__init__()
-        self.post_dumps.append(self.set_editable_post_dump)
+
+    can_edit = fields.Method("is_job_editable",)
```

Comment:
It will not work like that, because the Job will be indexed to elasticsearch with this field. We should either move it to the `ui` serializer (preferable) or leave it in the `post_dump`